xwiki-platform-core/xwiki-platform-filter/xwiki-platform-filter-ui/src/main/resources/Filter/FilterStreamJobJSON.xml

```diff
@@ -43,13 +43,9 @@
   #set($finished = $jobStatus.state.name() == 'FINISHED')
   #if ($finished)
     #if (!$jobStatus.logTail.hasLogLevel('ERROR'))
-      &lt;div class="box successmessage"&gt;
-        Done.
-      &lt;/div&gt;
+      #success('Done.')
```

Comment:
We're missing translations in that file it seems. Might worth adding a comment / creating ticket if there's none.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3555/commits/87fe4a4a9033dff7331aaa62f22753f75cdf524f :+1: